cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java

```diff
@@ -293,13 +234,13 @@ public ProgramLiveInfo getLiveInfo(ProgramId program) {
         for (Map.Entry<String, Collection<TwillRunResources>> entry : report.getResources().entrySet()) {
           for (TwillRunResources resources : entry.getValue()) {
             liveInfo.addContainer(new ContainerInfo(containerType,
-                                                    entry.getKey(),
-                                                    resources.getInstanceId(),
-                                                    resources.getContainerId(),
-                                                    resources.getHost(),
-                                                    resources.getMemoryMB(),
-                                                    resources.getVirtualCores(),
-                                                    resources.getDebugPort()));
+              entry.getKey(),
```

Comment:
these didn't need to get changed, it's fine for arguments to align on the next line.